disclosure_backend/views.py

```diff
@@ -45,6 +45,8 @@ def location_view(request, fips_id):
         required: true
     response_serializer: LocationSerializer
     """
+    # XXX: no-op so flake8 doesn't consider this an unused import
+    LocationSerializer()
```

Comment:
Good to know. I figured this will become a used import once we un-stub this API endpoint.